jlm/llvm/ir/operators/operators.hpp

```diff
@@ -58,13 +58,13 @@ class phi_op final : public jlm::rvsdg::simple_op
   inline const jlm::rvsdg::type &
   type() const noexcept
```

Comment:
This leads to a compilation error due to `-Wfatal-errors`. Moreover, there are more places where this could be applied. I think it should be done in another PR.